server/src/http/weatherApi.js

```diff
@@ -0,0 +1,17 @@
+// const $host = require('./index')
+const axios = require('axios');
+require('dotenv').config()
```

Comment:
Chekc if you really need to require 'dontenv' here, as you already do it on a top level